apps/jobs/views.py

```diff
@@ -2073,3 +2077,160 @@ def get_github_badge_data(
         else:
             data["message"] = f"{challenge_obj.title}"
     return Response(data, status=http_status_code)
+
+
+@api_view(["POST"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_presigned_url_for_submission(
+    request, challenge_pk, challenge_phase_pk
```

Comment:
Why do we need challenge_pk and challenge_phase_pk both? We can simply take and use challenge_phase_pk, no?